worksheet_B/README.md

```diff
@@ -0,0 +1,20 @@
+    List = Premade list of random words
+    Import Random
+    Import SequenceMatcher
+    randomWord = Random selects 1 word randomly from list
+    Tries = 0
+    While Tries < 4
+    Guess = Input
+    Input asks user to input a word
+    If Guess =/= randomWord Then
+    Run SequenceMatcher for word similarity
```

Comment:
The worksheet asked you to give pseudocode for the algorithm to compute the similarity, i.e. count the number of matching letters between the guess and the secret word. Note that Python's SequenceMatcher class doesn't do this -- it computes a much more complicated similarity metric.